docs/source/install.rst

```diff
@@ -5,44 +5,70 @@
 Installation
 ============
 
-The following is a quick start overview of installing the Jupyter Notebook and kernels. For detailed installation
-instructions for individual Jupyter or IPython projects, please see the documentation and/or GitHub repos
-of the individual projects listed :ref:`here <landing>`.
+The following is a quick start overview of installing the Jupyter Notebook and
+kernels. For detailed installation instructions for individual Jupyter or IPython
+subprojects, please see the documentation and/or GitHub repos of the individual
+projects listed :ref:`here <subprojects>`.
 
 Installing the Jupyter Notebook
 -------------------------------
 
-While the Jupyter Notebook allows users to run code in many different programming
-language, the Notebook itself is implemented in Python.
+While the Jupyter Notebook allows users to run code in many different
+programming language, the Notebook itself is implemented in Python. Thus to
+install the Notebook you will also need Python.
 
-**If you already have Python**, you can install the notebook using pip::
-
-    pip install jupyter
 
+If you are new to Python
+........................
```

Comment:
usually heading 3 are `~~~~` I don't really care which one but maybe we should be consistant across the projects. 

Thoughts >